orcid-core/src/main/java/org/orcid/core/togglz/Features.java

```diff
@@ -90,7 +93,6 @@
     @Label("Grouping suggestions")
     GROUPING_SUGGESTIONS,
     
-    @Label("Enable promotion of chosen orgs in search")
```

Comment:
@leomendoza123 Was this label intentionally removed?

Reply:
Thanks Liz I add it back